src/v/rpc/connection_cache.cc

```diff
@@ -240,7 +240,7 @@ ss::future<> connection_cache::apply_changes(
 }
 
 ss::future<> connection_cache::remove_broker_client_coordinator(
-  model::node_id self, model::node_id dest) {
```

Comment:
Any idea why this wasn't raising a warning on the previous compiler version? I feel like I see these get tripped all the time during development. Maybe it's always clang-tidy.

Reply:
> Any idea why this wasn't raising a warning on the previous compiler version? I feel like I see these get tripped all the time during development. Maybe it's always clang-tidy.

I don't know we have `-Wall -Wextra -Werror`, it';s been in `-Wextra` since before clang-16, I can't find a change.